sql/analyzer/analyzer_test.go

```diff
@@ -383,55 +325,31 @@ func TestMixInnerAndNaturalJoins(t *testing.T) {
 	expected := plan.NewProject(
 		[]sql.Expression{
 			expression.NewGetFieldWithTable(0, sql.Int32, "mytable", "i", false),
-			expression.NewGetFieldWithTable(4, sql.Float64, "mytable2", "f2", false),
+			expression.NewGetFieldWithTable(3, sql.Float64, "mytable2", "f2", false),
```

Comment:
No idea. @mcarmonaa is that from any of your PRs?

Reply:
Yes, it is. It's because now there aren't specific implementations for pushdown nodes that wrap the table node, all that is managed now by the `ResolvedTable` node. 

If there are projections and filters pushdowned, the `ResolvedTable` node offers the correct schema for the final result.